File: go/prep_data.go
package main

import (
	"fmt"
	"math/rand"
)

var num_range int = 10
var numbers []int

func main() {
	for i := 0; i < num_range; i++ {
		numbers = append(numbers, rand.Intn(1000))
	}
	fmt.Println("original")
	fmt.Println(numbers)
	fmt.Println("sorted")
}

File: python/prep_data.py
from random import randint

class PrepData:
    """
    Create list type data to be
    used in sorting programs.
    """

    def create_numbers(self):
        elements_number = randint(8, 10)
        numbers = [randint(0, 1000) for _ in range(elements_number)]
        return numbers

    def show_input_data(self):
        numbers = self.create_numbers()
        print('original')
        print(numbers)
        print('sorted')
        return numbers